agent-tooling/src/main/java/io/opentelemetry/auto/typed/base/BaseTypedTracer.java

```diff
@@ -26,32 +26,32 @@
   protected final Tracer tracer;
 
   protected BaseTypedTracer() {
-    tracer = OpenTelemetry.getTracerFactory().get(getInstrumentationName(), getVersion());
+    tracer = OpenTelemetry.getTracerProvider().get(getInstrumentationName(), getVersion());
   }
 
   protected abstract String getInstrumentationName();
 
   protected abstract String getVersion();
 
-  public final T startSpan(INSTANCE instance) {
+  public final T startSpan(final INSTANCE instance) {
```

Comment:
not sure why all these `final` crept into this PR.

Reply:
because the (unnamed 😄) person who added those files initially didn't follow https://github.com/open-telemetry/opentelemetry-auto-instr-java/blob/master/CONTRIBUTING.md

Reply:
at some point, we need to reconcile the coding standards between otel-java and the auto-instrumentation code. over in otel-java we don't final all the things. Also, I don't see parameters being final in that doc anywhere. Am I missing it?

Reply:
it's part of the save actions plugin (see "add final to ..." in the screenshot)

this was inherited from DataDog repo, so just going with that for now. definitely agree with sync'ing coding standards with otel-java 👍

Reply:
I don't see "add final to parameter" in there.  I wonder if IDEA considered parameters as local variables?

Reply:
Yeah, looks like that "save actions" plugin treats them the same.